src/components/Icons/Svg/FT/FTIcons.stories.tsx

```diff
@@ -22,6 +16,8 @@ const bscSymbolList = ['DAI', 'BNB', 'USDC', 'WETH', 'USDT'];
 
 const ethSymbolList = ['DAI', 'USDC', 'WETH'];
 
+const purrfectTokenList = ['PUR'];
```

Comment:
why you have a purrfectTokenList ?? purrfec is not a network

Reply:
it's just a mock